crates/uv/tests/common/mod.rs

```diff
@@ -253,8 +306,8 @@ pub fn bootstrapped_pythons() -> Option<Vec<PathBuf>> {
 
 /// Create a virtual environment named `.venv` in a temporary directory with the given
 /// Python version. Expected format for `python` is "python<version>".
-pub fn create_venv(
-    temp_dir: &assert_fs::TempDir,
+pub fn create_venv<Parent: assert_fs::prelude::PathChild + AsRef<std::path::Path>>(
+    temp_dir: &Parent,
```

Comment:
This is generic because some tests use a child directory to create a second virtual environment. I'm not excited about the approach in those tests, but we can address that later.